Desktop_Interface/isobuffer.cpp

```diff
@@ -2,391 +2,364 @@
 #include "isodriver.h"
 #include "uartstyledecoder.h"
 
-isoBuffer::isoBuffer(QWidget *parent, int bufferLen, isoDriver *caller, unsigned char channel_value) : QWidget(parent)
```

Comment:
Here and elsewhere:
It's safer to use the cstdint types such as uint8_t and int16_t, especially in a project such as this where bit positions matter.